server/database.js

```diff
@@ -92,11 +92,42 @@ const seedIntoDatabase = (function(data, callback) {
   })
 });
 
+const updateGame = (id, game, callback) => {
```

Comment:
if you rewrite this to use question marks and an array of query params, you can handle escaping.